src/main/java/org/folio/consortia/service/impl/PermissionUserServiceImpl.java

```diff
@@ -70,6 +70,12 @@ public void addPermissions(PermissionUser permissionUser, String permissionsFile
     });
   }
 
+  @Override
+  public void deleteUserPermissions(String userId) {
+    permissionsClient.deleteUser(userId);
+    log.info("deleteUserPermissions:: Deleting permissionUser with userId={}", userId);
```

Comment:
Deleting was already performed at this stage, so you can move this log before permissionsClient.deleteUser(userId);
or change Deleting to Deleted

Reply:
Fixed